src/components/layout/Sidebar.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Sidebar = props => (
-  <div className="col-xs-12 col offset-lg-1">
+  <aside className="col-xs-12 col offset-lg-1">
```

Comment:
👍 I've seen references to it, but haven't actually ever used the `aside` tag. TIL!

Reply:
Is the CSS not affected by this being an aside instead of a div? I'm guessing since it's probably referenced by class, ie `.col-xs-12` it is fine, but just curious

Reply:
Yeah, it's good for accessibility.

Reply:
Ah. I meant the CSS specs being affected by it being an aside instead of a div. I'm imagining we don't do this, but I was picturing some base level styling on all divs that we'd then be missing here since we're using an aside.